pkgs/applications/display-managers/lightdm-slick-greeter/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, vala
+, intltool
+, autoreconfHook
+, wrapGAppsHook
+, lightdm
+, gtk3
+, pixman
+, libcanberra
+, libX11
+, libXext
+, linkFarm
+, lightdm-slick-greeter
+, numlockx
+, numlockSupport ? false
```

Comment:
Why is this off by default? (TBH I will probably just drop this option and set that in `slick-greeter.conf` if numlockx does not have huge impact in closure so we don't have to build both when testing)